src/main/java/org/example/SpecificData.java

```diff
@@ -0,0 +1,33 @@
+package org.example;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class SpecificData {
+
+
+
+    public static void dis2(int i){
```

Comment:
Rename the method name and please follow standard naming pattern for all the variable, method and class names.